src/features/invoiceCard/InvoiceCard.tsx

```diff
@@ -57,6 +60,23 @@ const InvoiceCard = ({
         )
     );
 
+  const getSendButton = () => {
```

Comment:
Could be called `renderSendButton`, but I'll leave it up to you

Reply:
Changed.